### ISSUE TITLE
docker image is run by default as root (privileged) user

### ISSUE THREAD

**Issue by Constantin07** [terraform-linters/tflint-bundle#68]
At the moment docker image runs as `root` user when invoked.
I would suggest to use a non-privileged user in image by default.

**Comment by wata727** [comment]
You are right. But I'm a little concerned about the incompatibility of changing the default user...

**Comment by Constantin07** [comment]
@wata727 here is PR with the change https://github.com/terraform-linters/tflint-bundle/pull/69 ready for review please.

**Comment by wata727** [comment]
For example, the following code from Super-Linter will no longer work.
https://github.com/github/super-linter/blob/05f34c653f433dfac36856d446a0d624c46c2012/Dockerfile#L124

Originally, this image was not intended to be used as-is (probably too many things are missing), so a non-root user may not be necessary.
